docs/admin_tutorial.md

```diff
@@ -1,12 +1,14 @@
 # Admin tutorial
 
-This document provides an example on how to install and configure `pg_diffix` to expose a simple dataset
-for anonymized querying.
-It assumes that an existing installation of PostgreSQL 14 on a Linux system is available, a database
-named `test_db` was created, and the personal data is in the table `test_table`,
-containing a column named `id`, which uniquely identifies protected entities (the anonymization ID).
+This document provides examples on how to install and configure `pg_diffix` to expose a simple dataset for anonymized querying.  It assumes that an existing installation of PostgreSQL 14 on a Linux system is available.
 
-## Installation
+It provides multiple examples. The first is a [simple example](#simple-example) that assumes a database named `test_db` was created, and the personal data is in the table `test_table`, containing a column named `id`, which uniquely identifies protected entities (the anonymization ID).
+
+Following this are [several examples of real datasets](#real-examples) which discuss the rationale behind how to determine which columns contain anonymization IDs.
+
+# Simple Example
```

Comment:
We already have a top heading. This should have `##` and the following headings should have `###`.